priv/repo/seeds.exs

```diff
@@ -41,6 +42,9 @@ units_per_level = 5
 {:ok, _fertilizer_currency} =
   Users.Currencies.insert_currency(%{game_id: champions_of_mirra_id, name: "Fertilizer"})
 
+{:ok, _trophies_currency} =
+  Users.Currencies.insert_currency(%{game_id: curse_of_mirra_id, name: "Trophies"})
```

Comment:
FYI I don't think the seeds run when this is deployed, so you will not have this unless manually added or part of the migration

Reply:
right! i didn't though about that. I'll do the same as what the champions team is doing, running the seed manually when this is deployed to the server